src/tests/cli_perf.py

```diff
@@ -54,7 +55,7 @@ def setup():
     ret, out, err = run_proc(GPG, ['--batch', '--passphrase', '', '--homedir', GPGDIR, '--import', path.join(RNPDIR, 'pubring.gpg'), path.join(RNPDIR, 'secring.gpg')])
 
     # Generating small file for tests
-    SMALLSIZE = 3312;
+    SMALLSIZE = 33120;
```

Comment:
is it intended?

Reply:
Nope, changed during some testing. Thanks for noticing.